src/components/dev-hub/related-articles.js

```diff
@@ -42,23 +42,55 @@ const RelatedCards = styled('div')`
     }
 `;
 
+const getCardParamsFromRelatedType = (relatedArticle, slugTitleMapping) => {
+    console.log(relatedArticle);
+    const name = relatedArticle.name
+        ? relatedArticle.name
+        : relatedArticle.type;
+    switch (name) {
+        case 'doc':
+            const target = relatedArticle.target;
+            const slug = target && target.slice(1, target.length);
+            const image = relatedArticle.image || ARTICLE_PLACEHOLDER;
+            const title = dlv(slugTitleMapping, [slug, 0, 'value'], '');
```

Comment:
Not for now, but since we do this exact thing with dlv a bunch we should make a util function for this like `get-slug-title` or something

Reply:
Ah good point, I see what you mean by that now, thanks!